client/src/components/BecomeInterpreterForm.js

```diff
@@ -0,0 +1,167 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import { Button, Form, Dropdown } from "semantic-ui-react";
+import { Grid } from "semantic-ui-react";
+
+const languageOptions = [
+  { key: "Arabic", text: "Arabic", value: "Arabic" },
```

Comment:
All 3 properties are the same. You can make `key`s as something like `AR` for Arabic, `EN` for English, `RU` for Russian, etc.

Reply:
I changed the keys